tools/osbuilder/scripts/lib.sh

```diff
@@ -155,7 +154,7 @@ build_rootfs()
 	else
 		DNF="${DNF} --releasever=${OS_VERSION}"
 	fi
-	$DNF install ${EXTRA_PKGS} ${PACKAGES}
+	"$DNF" install "${EXTRA_PKGS}" "${PACKAGES}"
```

Comment:
I think this should be without quotes to allow to expand the args for dnf 
dnf install "bla bla bla" would be a argument but we need that each of them is one arg